test/testlocal/nft/anvil/AnvilFork.t.sol

```diff
@@ -134,16 +133,16 @@ contract AnvilFork is AnvilDeployment, Test {
       string memory cTypeName = erc20[cTypes[i]].symbol();
       vm.label(address(erc20[cTypes[i]]), cTypeName);
     }
-// #todo label the oracles by token; my suspicion is that they're in order with the first delayed oracle being wstETH
-//    for (uint256 i; i < denominatedOracles.length; i++) {
-//      string memory oracleName = denominatedOracles[i].symbol();
-//      vm.label(address(denominatedOracles[i]), oracleName);
-//    }
-//
-//    for (uint256 i; i < delayedOracles.length; i++) {
-//      string memory oracleName = delayedOracles[i].symbol();
-//      vm.label(address(delayedOracles[i]), oracleName);
-//    }
+    // #todo label the oracles by token; my suspicion is that they're in order with the first delayed oracle being wstETH
+    //    for (uint256 i; i < denominatedOracles.length; i++) {
+    //      string memory oracleName = denominatedOracles[i].symbol();
+    //      vm.label(address(denominatedOracles[i]), oracleName);
+    //    }
+    //
+    //    for (uint256 i; i < delayedOracles.length; i++) {
+    //      string memory oracleName = delayedOracles[i].symbol();
+    //      vm.label(address(delayedOracles[i]), oracleName);
+    //    }
```

Comment:
can you run `yarn lint:fix & yarn lint:sol` so these format changes don't happen

you can also set your editor to these, found in `foundry.toml` if it is more convenient:
```
[fmt]
line_length = 120
tab_width = 2
bracket_spacing = false
int_types = 'long'
multiline_func_header = 'params_first'
quote_style = 'single'
number_underscore = 'thousands'
wrap_comments = false
```

Reply:
ok, those lines are in the foundry.toml.  Although those changes were added when I ran the linter.  not sure why mine is acting odd.